core/schema-management.md

```diff
@@ -0,0 +1,40 @@
+# Schema Management
+
+## Problem Description
+
+To enable data exchange between business parties involved in a Baseline Workflow (or only Workstep?) (in short, parties), there must be present a set of pre-agreed schemas. 
+
+Furthermore, at least identifying different versions of the same schema must be possible for each Baseline instance. A Baseline instance may support different versions of the same schema.
```

Comment:
Might be good to elaborate on what a "baseline instance" is in this context.

Reply:
Agree, should I refer to this? [2.6 Baseline Protocol Instance](https://github.com/ethereum-oasis/baseline-standard/pull/50/files?short_path=6db9664#diff-6db9664e3b55a0ca9841f1be1ca73aec567763fba50a0cc4dcaa4514ec88be4d)

I think I need to wait for other PR to be merged and use the proper link.